sdk/test/NetStandard/IntegrationTests/IntegrationTests/DynamoDB/DataModelTests.cs

```diff
@@ -383,7 +383,7 @@ private async Task TestHashObjects()
                 Assert.Equal(product.KeySizes.Count, retrieved.KeySizes.Count);
                 Assert.NotNull(retrieved.CompanyInfo);
                 Assert.Equal(product.CompanyInfo.Name, retrieved.CompanyInfo.Name);
-                Assert.Equal(product.CompanyInfo.Founded, retrieved.CompanyInfo.Founded);
+                Assert.Equal(product.CompanyInfo.Founded.ToUniversalTime(), retrieved.CompanyInfo.Founded);
```

Comment:
is this one of the tests that were failing b/c CompanyInfo.Founded was returning a local time?

Reply:
Yes, this was the failing test. However if you look at this commit: https://github.com/aws/aws-sdk-net/commit/5f8866085fb055d8333432804a2ff5cb2e8730e8 which is part of this PR you will see where I switched the default to UTC. Because of this UTC is being returned and the product.CompanyInfo.Founded is in unspecified time. So this is now updated to ToUniversal only for the assert to pass.